packages/admin/admin/src/appHeader/menuButton/AppHeaderMenuButton.tsx

```diff
@@ -11,9 +11,19 @@ export type AppHeaderMenuButtonClassKey = IconButtonClassKey;
 
 export const AppHeaderMenuButton = (inProps: AppHeaderMenuButtonProps) => {
     const { children: propChildren, ...restProps } = useThemeProps({ props: inProps, name: "CometAdminAppHeaderMenuButton" });
-    const { toggleOpen, open } = useContext(MenuContext);
+    const { toggleOpen, open, drawerVariant } = useContext(MenuContext);
 
-    const children = !propChildren ? open ? <HamburgerClose fontSize="large" /> : <HamburgerOpen fontSize="large" /> : propChildren;
+    const closeIcons: Record<typeof drawerVariant, ReactNode> = {
+        temporary: <Close />,
+        permanent: <HamburgerClose fontSize="large" />,
+    };
+
+    const openIcons: Record<typeof drawerVariant, ReactNode> = {
+        permanent: <HamburgerOpen fontSize="large" />,
+        temporary: <Hamburger />,
```

Comment:
```suggestion
        temporary: <Hamburger />,
        permanent: <HamburgerOpen fontSize="large" />,
```

That's a nit for sure 😂

Reply:
Haha I get it though. If I would've review this, that would've also triggered me^^. I don't even know how my perfectionist brain didn't get that 🤣